packages/teams-js/src/public/chat.ts

```diff
@@ -107,7 +109,7 @@ export namespace chat {
           user: openChatRequest.users[0],
           message: openChatRequest.message,
         };
-        openChat(chatRequest);
+        resolve(openChat(chatRequest));
```

Comment:
>resolve [](http://example.com/codeflow?start=8&length=7)

Without this, calling `openGroupChat` with a single user would cause `openGroupChat` to return immediately, not waiting for the chat message to be received and acknowledged by the host.